### DISCUSSION TITLE
Pertaining pull request policy...

### DISCUSSION THREAD

**Discussion by vst-name** [directus-labs/helm-chart#27] [General]
Hello, I'm looking to contribute with regards to DB driver options, are there any policies regarding pull requests?

I've also noticed that Image isn't quite up to date with the latest, is there no git actions for auto update, or is there a reason to keep it more stable?

Thank you.

**Comment by mykhailo-sindieiev** [comment]
Hello there! 

Unfortunately, we do not have PR template yet. Feel free to open a freestyle PR with any changes you think is useful. 

Regarding image tags. 

I am working on my spare time on this repo and dependabot workflow is pending.

Any help is appreciated
